dataverse-webapp/src/main/webapp/resources/vecler/theme.scss

```diff
@@ -2454,7 +2529,1175 @@ div.ui-dialog.largePopUp {
+		text-decoration: underline;
+
+		&:active, &:hover, &:focus {
+			color: $contrast-foreground-color;
+			text-decoration: underline;
+		}
+	}
+
+	.dropdown-menu > li > a,
+	.dropdown-menu > form > li > a,
+	#navbarFixed .dropdown.accountName .logout a {
+		text-decoration: underline;
+
+		&:active, &:hover, &:focus {
+			background-color: $contrast-hover-color !important;
+			text-decoration: underline;
+		}
+	}
+
+	div[id$="notifications"].ui-tabs-panel div.notification-item-cell,
+	.panel-default > .panel-heading {
+		border-bottom-color: $contrast-link-color;
+	}
+
+	hr, .ui-widget-header, .button-block.form-top, .form-field-group {
+		border-color: $contrast-link-color;
+	}
+
+	.ui-state-hover, .ui-widget-content .ui-state-hover, .ui-widget-header .ui-state-hover {
+		background-color: $contrast-hover-color;
+	}
+
+	/* Icons */
+
+	#ajaxStatusPanel_start img {
+		filter: $contrast-ajax-spinner-filter;
+	}
+
+	[data-icon]::before, .icon-astro::before, .icon-audio::before, .icon-code::before, .icon-dataset::before, .icon-dataverse::before, .icon-dataverse::after, .icon-document::before, .icon-file::before, .icon-geodata::before, .icon-image::before, .icon-network::before, .icon-other::before, .icon-package::before, .icon-tabular::before, .icon-unlock::before, .icon-video::before,
+	#dv-sidebar a.facetTypeLink {
+		color: $contrast-foreground-color;
+	}
+	.icon-dataverse::before {
+		background: none;
+	}
+
+	.ui-message, .ui-messages, * {	
+		.ui-message-info, .ui-message-warn, .ui-message-error, .ui-message-fatal,
+		.ui-messages-info, .ui-messages-warn, .ui-messages-error, .ui-messages-fatal {
+			color: $contrast-foreground-color;
+
+			.ui-icon {
+				background-image: url($contrast-primefaces-main);
+			}
+		}
+		
+		.ui-message-info-icon::before,
+		.ui-messages-info-icon::before,
+		.ui-message-warn-icon::before,
+		.ui-messages-warn-icon::before,
+		.ui-message-error-icon::before,
+		.ui-messages-error-icon::before,
+		.ui-message-fatal-icon::before,
+		.ui-messages-fatal-icon::before {
+			color: $contrast-foreground-color;
+		}
+	}
+
+	.ui-state-default .ui-icon, .ui-widget-header .ui-icon, .ui-widget-content .ui-icon {
+		background-image: url($contrast-primefaces-main);
+	}
+
+	/* Alerts, errors, notifications */
+
+	.alert {
+		color: $contrast-foreground-color;
+	
+		&.alert-info, &.alert-success, &.alert-danger, &.alert-warning {
+			background: $contrast-background-color;
+			border-color: $contrast-foreground-color;
+		}
+	}
+
+	.text-warning {
+		color: $contrast-link-color;
+	}
+	.text-danger {
+		color: $contrast-alt-focus-color;
+	}
+	.text-success {
+		color: $contrast-foreground-color;
+	}
+
+	.bg-danger {
+		border-color: $contrast-foreground-color;
+		background: $contrast-background-color;
+
+		.text-danger {
+			color: $contrast-foreground-color;
+		}
+	}
+
+	.ui-autocomplete-emptyMessage {
+		border-color: $contrast-foreground-color;
+		background: $contrast-background-color;
+		color: $contrast-foreground-color;
```

Comment:
Border for empty autocomplete is still not showing up
![autocomplete_border_1](https://user-images.githubusercontent.com/8190961/78988696-f9a25200-7b31-11ea-81f2-2afe73ae96d3.png)
maybe we should add `border-style: solid;`?
![autocomplete_border](https://user-images.githubusercontent.com/8190961/78988715-03c45080-7b32-11ea-85f1-dd8faed94606.png)

Reply:
Yeah, I overlooked that. Fixed